hack/tools/deployment/deployment-update.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+var (
+	ValidEnvironments = []string{
```

Comment:
this could be a map[string][]string with the env as the hash key and sectors list as the value. Then there's no need to switch case below